assets/js/TbaApi.js

```diff
@@ -1,5 +1,7 @@
 const tbaUrl = "https://www.thebluealliance.com/api/v3";
-const apiKey = "Not Here";
+const apiKey = "FglzQyQwA2VhUenZ1BcfuFGR2xA6lVWMtyvVaOjMTfjJUgz1ozF6j9GO71GlirLV";
```

Comment:
Oops, your secret isn't so secret now. Let's not do this until you've got a good mechanism to keep this secret a secret. Oh, and change the value of the secret for your own sake.

Reply:
Good catch, fixed it and switched it. https://github.com/FRCTeam1987/teambroncobots.com/commit/c89c13e3c3bca1a6cbe3fb146e16d839c0ea25a0